### PR TITLE
{vis}[GCCcore/6.4.0] SDL2_image v2.0.3

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — DISMISSED** [review]

**Comment by migueldiascosta** [comment]
@JackPerdue, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

File: easybuild/easyconfigs/l/libxml2/libxml2-2.9.8-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name = 'libxml2'
+version = '2.9.8'
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/l/libxml2/libxml2-2.9.8-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL2/SDL2-2.0.8-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SDL2'
+version = '2.0.8'
+
+homepage = 'http://www.libsdl.org/'
+
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL2/SDL2-2.0.8-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL2_image/SDL2_image-2.0.3-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+easyblock = 'ConfigureMake'
+
+name = 'SDL2_image'
+version = '2.0.3'
+
+homepage = 'https://www.libsdl.org/projects/SDL_image/'
+
+description = """
+ SDL_image is an image file loading library. 
+"""
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL2_image/SDL2_image-2.0.3-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL2/SDL2-2.0.8-GCCcore-6.4.0.eb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+description = "SDL: Simple DirectMedia Layer, a cross-platform multimedia library"
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+# http://www.libsdl.org/release/SDL2-2.0.8.tar.gz
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL2/SDL2-2.0.8-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL2_image/SDL2_image-2.0.3-GCCcore-6.4.0.eb [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+# https://www.libsdl.org/projects/SDL_image/release/SDL2_image-2.0.3.tar.gz
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL2_image/SDL2_image-2.0.3-GCCcore-6.4.0.eb]: Done

File: easybuild/easyconfigs/s/SDL2_image/SDL2_image-2.0.3-GCCcore-6.4.0.eb [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+dependencies = [
+    ('libjpeg-turbo', '1.5.3'),
+    ('libpng', '1.6.34'),
+    ('LibTIFF', '4.0.9'),
+    ('SDL2', '2.0.8'),
+]
+
+sanity_check_paths = {
+    'files': ['include/SDL2/SDL_image.h', 'lib/libSDL2_image.so',
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/s/SDL2_image/SDL2_image-2.0.3-GCCcore-6.4.0.eb]: Done

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/4d746b1326b31b499bacc91af0576b64 for a full test report.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**